html/semantics/interactive-elements/the-dialog-element/centering.html

```diff
@@ -42,7 +55,19 @@
     iframe.height = iframeHeight;
     iframe.onload = t.step_func_done(() => {
       const dialog = iframe.contentDocument.querySelector("dialog");
-      assert_equals(iframe.contentWindow.getComputedStyle(dialog)[property], numericValue + "px");
+      const dialogStyle = iframe.contentWindow.getComputedStyle(dialog);
+      assert_equals(dialogStyle[property], numericValue + "px");
+      assert_equals(dialogStyle['inset-inline-start'], "0px");
+      assert_equals(dialogStyle['inset-inline-end'], "0px");
```

Comment:
I think I'm missing something, but how can the style have changed between these asserts and those below? Or if they're all indeed 0px, what are we testing?

Reply:
Er, they aren't going to change. We are testing they are set to `0px` and equals to the corresponding `top/bottom/left/right`.

Reply:
In that case, would it be clearer to rewrite the below as follows?

```js
      if (dialogWM.startsWith("vertical")) {
        assert_equals(dialogStyle['top'], "0px");
        assert_equals(dialogStyle['bottom'], "0px");
      } else {
        assert_equals(dialogStyle['left'], "0px");
        assert_equals(dialogStyle['right'], "0px");
      }
```

But that leaves me confused how this interacts with cases where `property` is `top` and `dialogWM.startsWith("vertical")`. Then you're testing that `dialogStyle['top']` is both `numericValue` and `0`?

Reply:
I think for that case, we shouldn't test `property` as `top`, because as you just said, it doesn't make sense. 

Cross-referencing the [other changes](https://github.com/web-platform-tests/wpt/pull/23484/files) that I think I plan to make to this file. One thing I changed was to not test `top` for vertical writing mode.